OpenEdX/AppDelegate.swift

```diff
@@ -35,13 +32,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
     ) -> Bool {
         initDI()
-        
         if let config = Container.shared.resolve(ConfigProtocol.self) {
             Theme.Shapes.isRoundedCorners = config.theme.isRoundedCorners
-            if let configuration = config.firebase.firebaseOptions {
-                FirebaseApp.configure(options: configuration)
-                Crashlytics.crashlytics().setCrashlyticsCollectionEnabled(true)
-            }
```

Comment:
We can't remove that part of the code, Crashlytics will stop working.

Reply:
@volodymyr-chekyrta We've added [FirebaseCrashlyticsCollectionEnabled ](https://github.com/openedx/openedx-app-ios/pull/291/files#diff-e70052e8a1bf7ccec67056e9557328a349fe3aa78447addbc7d65542d6515c08R32) in the info.plist, I believe this will do the job.